src/components/Section/Relationships/People/Person.jsx

```diff
@@ -174,7 +174,7 @@ export default class Person extends React.Component {
             className="rank-notapplicable"
             {...this.props.RankNotApplicable}
             label={i18n.t(
-              'relationships.people.person.label.notApplicable'
+              'relationships.people.person.label.emailNotApplicable'
```

Comment:
I think this has ventured in the realm of bike-shedding variable names, which I want to encourage us to avoid. And in this case, the original label name seemed to be consistent:
* The label is for a NotApplicable component
* The property associated with the label is called EmailNotApplicable
* The "I don't know" value is a localization/presentation view of this particular NotApplicable component. In general (but not likely on this form), a NotApplicable component could have label values like "我不知道" or "no lo sé", and those wouldn't necessarily change how we name concepts in the implementation/programmer view.

No changes requested at this point (unless you personally wanted to revert last commit), I just want to set the threshold for variable name changing a little higher. Thanks!

Reply:
Thanks for the context. If there aren't required changes, I prefer to leave my changes for consistency since other locale files use `idk` as the label. (e.g. `foreign.contacts.label.idk`, `foreign.business.sponsorship.label.idk`, `reference.label.`, `legal.investigations.history.label.idk`, etc.)